pwnlib/util/cyclic.py

```diff
@@ -209,6 +209,7 @@ def cyclic_find(subseq, alphabet = None, n = None):
 
     if isinstance(subseq, six.integer_types):
         subseq = packing.pack(subseq, bytes=n)
+    subseq = context._encode(subseq)
```

Comment:
This breaks python 3 build. What is your usecase?

Reply:
I hope `cyclic_find` can also accept a str likes `cyclic_find('baaacaaa')` besides of `cyclic_find(b'baaacaaa')` in Python 3.

Reply:
This must be done somehow differently. `_encode` enforces bytes, while we need the type that is used in the alphabet. The internal library usage does not use `str` with binary alphabets, nor does it use `bytes` with text alphabets.

Reply:
I think we can unify `subseq` and `alphabet` to `bytes`. Add a commit to fix the build.